content/actions/use-cases-and-examples/building-and-testing/building-and-testing-rust.md

```diff
@@ -85,58 +117,57 @@ jobs:
       matrix:
         BUILD_TARGET: [release] # refers to a cargo profile
     outputs:
-      release_built: ${{ steps.set-output.outputs.release_built }}
+      release_built: {% raw %}${{ steps.set-output.outputs.release_built }}{% endraw %}
     steps:
       - uses: {% data reusables.actions.action-checkout %}
-      - name: Build binaries in "${{ matrix.BUILD_TARGET }}" mode
-        run: cargo build --profile ${{ matrix.BUILD_TARGET }}
-      - name: Run tests in "${{ matrix.BUILD_TARGET }}" mode
-        run: cargo test --profile ${{ matrix.BUILD_TARGET }}
+      - name: Build binaries in "{% raw %}${{ matrix.BUILD_TARGET }}{% endraw %}" mode
+        run: cargo build --profile ${% raw %}{{ matrix.BUILD_TARGET }}{% endraw %}
+      - name: Run tests in "${% raw %}{{ matrix.BUILD_TARGET }}{% endraw %}" mode
+        run: cargo test --profile ${% raw %}{{ matrix.BUILD_TARGET }}{% endraw %}
+```
+
+The `release` keyword used in this example corresponds to a cargo profile. You can use any [profile](https://doc.rust-lang.org/cargo/reference/profiles.html) you have defined in your `Cargo.toml` file.
+
+## Publishing your package or library to crates.io
```

Comment:
I'm assuming that this is the most common use case, so I moved it so that it was before the section on uploading artifacts to GitHub.